pkg/server/proto/gateway.yml

```diff
@@ -547,3 +547,13 @@ http:
 
   - selector: hashicorp.waypoint.Waypoint.GetTFCRunStatus
     get: /project/{project.project}/tfcrunstatus
+
+    # Events
+  - selector: hashicorp.waypoint.Waypoint.UI_ListEvents
+    get: /ui/project/{application.project}/application/{application.application}/events
+
+  - selector: hashicorp.waypoint.Waypoint.UI_ListEvents
+    get: /ui/project/{application.project}/application/{application.application}/workspace/{workspace.workspace}/events
+
+  - selector: hashicorp.waypoint.Waypoint.INTERNAL_AdditionalMessages
+    get: /ui/events/event-schemas
```

Comment:
can we modify this name to also be internal additional messages instead of event schemas?

Reply:
ahh, yes I forgot about that. yeah I'll update that

Reply:
After that, merge away!

Reply:
done